fileversion.go

```diff
@@ -286,6 +287,9 @@ func (self *FileVersion) GetFirstStringFileInfo() (*StringFileInfo, error) {
 	if err != nil {
 		return nil, err
 	}
+	if len(tr) == 0 {
+		return nil, errors.New("Translations is empty");
```

Comment:
[Nitpick] Don't need the semicolon here.  Also, the Go convention is for error messages to start with a lowercase letter.  Something like `"no translations"` would be good.

Reply:
You right, corrected.